examples/spec/integration/register_namespace_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'workflows/timeout_workflow'
+
+describe 'Temporal.register_namespace' do
+  it 'can register a new namespace' do
```

Comment:
This tests 4 separate things. Why not break these into individual tests so we can confirm a) it's found, b) the name is correct, c) the data matches, and d) the retention seconds are correct?